src/test/java/org/greenbuttonalliance/gbaresourceserver/usage/repository/ElectricPowerQualitySummaryRepositoryTest.java

```diff
@@ -80,8 +80,7 @@ public void findAll_returnsAll() {
 //	public void entityMappings_areNotNull() {
 //	}
 
-	// encapsulated in a method to make available to other test classes
-	public static List<ElectricPowerQualitySummary> buildTestData() {
```

Comment:
Since we discussed that each repository test class should hard-code its test data instead of calling build methods of other classes, I changed the access modifiers on all these methods to `private`

Reply:
Thanks for doing all the refactoring.